### ISSUE TITLE
Github actions build broken

### ISSUE THREAD

**Issue by sfuhrm** [sfuhrm/openssl4j#6]
```
# Created at 2023-05-28T10:49:34.029
/opt/hostedtoolcache/Java_Adopt_jdk/11.0.19-7/x64/bin/java: symbol lookup error: /tmp/native1164465476988834680/libopenssl4j-Linux-amd64.so: undefined symbol: EVP_CIPHER_do_all

```

**Comment by sfuhrm** [comment]
Problem mitigated by moving experimental cipher implementation to branch CIPHER. See #3

**Comment by sfuhrm** [comment]
Branch CIPHER fixed with #7
